goconf/users/urls.py

```diff
@@ -9,7 +9,7 @@
 
 app_name = "users"
 urlpatterns = [
-    path("", view=user_list_view, name="list"),
+    path("", view=conf_list, name="conf_list"),
```

Comment:
It's not needed here. All urls in goconf/users app have the `/users/` prefix

Reply:
what exactly is not needed?